test/visual.test.js

```diff
@@ -0,0 +1,82 @@
+import Promise from 'bluebird';
+import test from 'tape';
+import msonZoo from 'mson-zoo';
+import webdriverio from 'webdriverio';
+import webdrivercss from 'webdrivercss';
+
+// Setup webdriver tests
+// We are using older webdrivercss, because wdio regression tests seems not ready yet
+const client = webdriverio.remote({ desiredCapabilities: { browserName: 'chrome' } });
+
+webdrivercss.init(client, {
+  screenshotRoot: 'visual-regressions',
+  failedComparisonsRoot: 'visual-regressions/failed',
+  misMatchTolerance: 0.05, // To prevent failures from font-rendering issues and small deviations
+  updateBaseline: false, // You can update baseline tests here
```

Comment:
Added the command. Also, we could plug this in if we find out we do plenty of changes https://github.com/webdriverio/webdrivercss-adminpanel